app/scripts/lib/validate.js

```diff
@@ -103,7 +103,41 @@ define([
       }
 
       return password.length >= Constants.PASSWORD_MIN_LENGTH;
+    },
+
+    /**
```

Comment:
My initial thought is this logic belongs in a different module. Validate's intent is to validate whether a value/set of values is actually valid. These functions choose a value from one or more values.